format_me.sh

```
@@ -1,5 +1,5 @@
-find . -iname "*.cpp" -not -path "*build*" -not -path "*third_party*" -exec clang-format -style=file -i {} \;
-find . -iname "*.hpp" -not -path "*build*" -not -path "*third_party*" -exec clang-format -style=file -i {} \;
-find . -iname "*.cc" -not -path "*build*" -not -path "*third_party*" -exec clang-format -style=file -i {} \;
-find . -iname "*.cxx" -not -path "*build*" -not -path "*third_party*" -exec clang-format -style=file -i {} \;
-find . -iname "*.h" -not -path "*build*" -not -path "*third_party*" -exec clang-format -style=file -i {} \;
\ No newline at end of file
+find . -iname "*.cpp" -not -path "*build*" -not -path "*third-party*" -exec clang-format -style=file -i {} \;
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)